tests/kafkatest/services/verifiable_producer.py

```diff
@@ -104,6 +100,10 @@ def _worker(self, idx, node):
         log_config = self.render('tools_log4j.properties', log_file=VerifiableProducer.LOG_FILE)
         node.account.create_file(VerifiableProducer.LOG4J_CONFIG, log_config)
 
+        # Configure security
+        self.security_config = self.kafka.security_config.client_config(node=node)
+	self.security_config.setup_node(node)
```

Comment:
@ijuma Thank you for the review. I had used tab instead of space, fixed and started a new system test run: https://jenkins.confluent.io/job/system-test-kafka-branch-builder/672/